src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PSScorer.java

```diff
@@ -25,6 +25,8 @@
 public final class PSScorer {
 
     public static final String HITS_TAG = "YP";
+    final static String KINGDOM_RANK_NAME = "kingdom";
+    final static String SUPERKINGDOM_RANK_NAME = "superkingdom";
```

Comment:
There is already a bunch of constants that seem related like DEFAULT_KINGDOM_ID, ROOT_ID, VIRUS_ID...KINGDOM_RANK_NAME SKINGDOM_RANK_NAME... would it make sense to collect them in a common Class/location?

Reply:
Moved the ones that pertain to NCBI taxonomy database constants to a new class PSTaxonomyConstants. I think the others are specific to their classes and should stay there.